src/users/users.controller.ts

```diff
@@ -1,25 +1,34 @@
 import { Controller, Get, Param, Request, UseGuards } from "@nestjs/common";
 import { UsersService } from "./users.service";
-import { ApiBearerAuth, ApiTags } from "@nestjs/swagger";
+import { ApiBearerAuth, ApiOperation, ApiTags } from "@nestjs/swagger";
 import { JwtAuthGuard } from "../auth/jwt-auth.guard";
 
 @Controller("users")
 @ApiTags("users")
 export class UsersController {
     constructor(private readonly usersService: UsersService) {}
 
+    @ApiOperation({
+        summary: "Gets all users.",
+    })
     @Get()
     findAll() {
         return this.usersService.findAll();
     }
 
+    @ApiOperation({
+        summary: "Gets a unique user on login.",
```

Comment:
I'd probably say gets logged in user detail (?)

Reply:
I updated to say gets logged in user detail but added description so it's known that this one uses the jwt token to pull the userId since it's close similar to the getUserDetailsById but the parameters are different.